src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1273,6 +1273,7 @@ public static partial class GC
         public static GCMemoryInfo GetGCMemoryInfo() { throw null; }
         public static int GetGeneration(object obj) { throw null; }
         public static int GetGeneration(System.WeakReference wo) { throw null; }
+        public static long GetTotalAllocatedBytes(bool precise = false) { throw null; }
```

Comment:
Can you add some basic tests for this?

Reply:
Yes. Will add. 
We have tests on the coreclr side, but having something trivial in CoreFX would be useful too.